.github/workflows/healthcheck.yml

```diff
@@ -0,0 +1,28 @@
+name: Health check
+on:
+  push:
+    branches:
+      - main
```

Comment:
@biglovisa what will run this health check everyday?

Reply:
updated it to be a cron!